client/stats/host.go

```diff
@@ -117,39 +116,45 @@ func (h *HostStatsCollector) collectLocked() error {
 	// Determine up-time
```

Comment:
Should this and `Collect()` still return an error?  May make sense to make them void functions instead?  Simplifies `client.go` implementation.

Reply:
Yeah - I'm planning on going back and refactoring this a little bit next week. Mostly wanted to get something together to help folks with debugging.